pom.xml

```
@@ -3,7 +3,7 @@
     <parent>
         <groupId>com.fasterxml.jackson</groupId>
         <artifactId>jackson-base</artifactId>
-        <version>2.14.0-rc2</version>
+        <version>2.14.2</version>
```

The latest as of today:

```suggestion
        <version>2.17.2</version>
```